contracts/test/integration/SubnetActorDiamond.t.sol

```diff
@@ -807,76 +811,6 @@ contract SubnetActorDiamondTest is Test, IntegrationTestBase {
         }
     }
 
-    function testSubnetActorDiamond_submitBottomUpMsgBatch_basic() public {
```

Comment:
I don't know if this is currently but if not, it would be good to have a test message for the happy path of message execution and with some of the failure scenarios that we had before for the message batch code.